<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-01Q950H6" failures="5" tests="7" name="Test" time="67.272" errors="0" timestamp="2024-10-23T10:50:11 IST">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_account_registration" time="2.19"/>
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="1.588"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.485">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="10.099">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="10.124">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="10.08">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="10.12">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test -->
